src/quicktype-core/language/Elm.ts

```diff
@@ -625,26 +625,26 @@ import Dict exposing (Dict, map, toList)`);
 
         this.forEachTopLevel(
             "leading-and-interposing",
-            this.emitTopLevelDefinition,
+            (t: Type, topLevelName: Name) => this.emitTopLevelDefinition(t, topLevelName),
```

Comment:
I don't usually declare argument types in these anonymous functions.  In some cases TypeScript can't infer them, but those are rare.  No big deal, doesn't need correction.  The general idea is that I want to declare types at interface boundaries to avoid type drift.  In this case here I just care that the types are compatible, not what they are in detail.

Reply:
Good to know!